changelog.json

```
@@ -1,5 +1,27 @@
 {
   "releases": {
+    "2.4": [
+      "[New] Set Git proxy environment variables from system configuration automatically - #9154",
+      "[Added] Enable creating a GitHub issue from app menu and keyboard shortcut - #8989",
+      "[Added] Add editor support for IntelliJ Idea on macOS - #8398. Thanks @gssbzn!",
+      "[Added] Add editor support for PhpStorm and update WebStorm - #7195. Thanks @dacoto97!",
+      "[Added] Add shell support for Windows Terminal - #7854. Thanks @AndreiMaga!",
+      "[Added] Support keyboard scrolling in diffs - #9074",
+      "[Fixed] Ensure branch protection warning works when branch names contain special characters - #9086",
+      "[Fixed] Ensure intro page is responsive when resizing the window - #9175",
+      "[Fixed] Ensure local branch is also deleted when deleting a branch that no longer exists on the remote - #8831",
+      "[Fixed] Fix graphic indicating files selected in Dark Theme - #8716. Thanks @say25!",
```

```suggestion
      "[Fixed] Display correct graphic indicating files selected in Dark Theme - #8716. Thanks @say25!",
```

---

looked at this PR again, it was the right graphic but missing some negative space within it. i'm not sure what the right verb would be here... maybe this?
```suggestion
      "[Fixed] Restore negative space in \"Files selected\" graphic for Dark Theme - #8716. Thanks @say25!",
```

---

Eh, I don't care what verb we use, I think `restore negative space` is fine! I just have a personal aversion to having [Fixed] issues start with Fix. That's probably a silly line to have though. 👍

---

@nerdneha totally! i appreciate you calling this line out because i missed the duplicate “fix” and would also like to avoid those.